### PR TITLE
Add oVirt ovirt_permissions and ovirt_permissions_facts modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

- ovirt_permissions
- ovirt_permissions_facts
##### SUMMARY

This PR adds two new modules for managing permissions in oVIrt.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_permissions_facts.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            - "Namespace of the authorization provider, where user/group resides."
+        required: false
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all permissions of user john:
```
Comment (mwperina) [inline cloud/ovirt/ovirt_permissions_facts.py]: I'd rephrase: 'Gather facts about all permissions of user with username C<john>:'

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! :+1:

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_permissions.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
```
Comment (ryansb) [inline cloud/ovirt/ovirt_permissions.py:23]: Same comment - please look at how the AWS modules handle these imports to surface errors to users.

**Comment by ryansb** [comment]
Looks good, other than the import statement

**Comment by machacekondra** [comment]
shipit

**Review by mwperina — APPROVED** [review]
:+1:
